src/pages/edge/edge-network/datastream-config-override-using-tags.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Datastream config override for Edge events using Tags.
+description: This guide provides instructions on overriding datastream ID and datastream configuration for Edge Network Requests that are not initiated by the sendEvent API, all accomplished through Tags Rules.
+keywords:
+- Edge Network
+- Datastream config overrides
+- Datastream ID override
+- Guide
+---
+
+# Overriding Datastream ID and Configuration for Edge Network Requests Using Tags Rules
+
+<InlineAlert variant="info" slots="text"/>
+
+Starting with version **2.4.0** of the `Edge` Android extension and version **4.3.0** of the `AEPEdge` iOS extension, the `sendEvent` API now supports optional Datastream overrides. For detailed information, please refer to the [sendEvent API reference](./api-reference.md#sendEvent).
```

Comment:
Starting with `Edge` extension version **2.4.0** (Android) and **4.3.0** (iOS), the `sendEvent` API supports ...

Reply:
Under this top header can you give a short intro that by default all the Edge requests are sent to the datastream configuration set in the Adobe Experience Platform Edge Network configuration settings and this tutorial provides examples on how to override it per event.